src/service-providers/active_campaign/ProviderSidebar.js

```diff
@@ -146,57 +218,69 @@ const ProviderSidebarComponent = ( {
+				onChange={ value => updateMeta( { ac_from_name: value } ) }
 			/>
 			<TextControl
 				label={ __( 'Email', 'newspack-newsletters' ) }
 				className="newspack-newsletters__email-textcontrol"
 				value={ acData.ac_from_email }
 				type="email"
 				disabled={ inFlight }
-				onChange={ value => updateMetaValue( 'ac_from_email', value ) }
+				onChange={ value => updateMeta( { ac_from_email: value } ) }
 			/>
 			<hr />
 			<strong className="newspack-newsletters__label">
 				{ __( 'Send to', 'newspack-newsletters' ) }
 			</strong>
-			<BaseControl className="newspack-newsletters__list-select">
-				<SelectControl
-					label={ __( 'To', 'newspack-newsletters' ) }
-					value={ acData.ac_list_id }
-					options={ [
-						{
-							value: '',
-							label: __( '-- Select a list --', 'newspack-newsletters' ),
-						},
-						...lists.map( ( { id, name } ) => ( {
-							value: id,
-							label: name,
-						} ) ),
-					] }
-					onChange={ value => updateMetaValue( 'ac_list_id', value ) }
-					disabled={ inFlight }
-				/>
-				{ acData.ac_list_id && (
-					<SelectControl
-						label={ __( 'Segment', 'newspack-newsletters' ) }
-						value={ acData.ac_segment_id }
-						options={ [
-							{
-								value: '',
-								label: __( '-- Select a segment (optional) --', 'newspack-newsletters' ),
-							},
-							...segments.map( ( { id, name } ) => ( {
-								value: id,
-								label: name,
-							} ) ),
-						] }
-						onChange={ value => updateMetaValue( 'ac_segment_id', value ) }
-						disabled={ inFlight }
+			<SendTo
+				availableItems={ lists }
+				onChange={ selected => onChangeSendTo( selected ) }
+				placeholder={ __( 'Type a list name to search', 'newspack' ) }
+				reset={ async () => {
+					updateMeta( { ac_list_id: '' } );
+					updateMeta( { ac_segment_id: '' } );
+				} }
+				selectedItem={ selectedList }
+			/>
+			{ selectedList && (
+				<>
+					<SendTo
+						availableItems={ segments }
+						onChange={ selected => onChangeSendTo( selected, 'segment' ) }
+						formLabel={ __( 'Filter by segment (optional)', 'newspack' ) }
+						placeholder={ __( 'Type a segment name to search', 'newspack' ) }
+						reset={ async () => updateMeta( { ac_segment_id: '' } ) }
+						selectedItem={ selectedSegment }
 					/>
-				) }
-				{ inFlight && <Spinner /> }
-			</BaseControl>
+				</>
+			) }
+			{ selectedList && (
+				<p
+					dangerouslySetInnerHTML={ {
+						__html: sprintf(
+							// Translators: %1$s is the number of members, %2$s is the item name, %3$s is the item type, %4$s is the parent item name and type (if any).
+							__(
+								'This newsletter will be sent to all %1$s in the %2$s %3$s%4$s.',
+								'newspack-newsletters'
+							),
+							selectedSegment?.details || ( ! selectedSegment && selectedList?.details )
+								? `<strong>${ selectedSegment?.details || selectedList?.details }</strong> `
+								: __( 'contacts', 'newspack-newsletters' ),
```

Comment:
Actually, looking at this again, this is not a very good way to localize the string.

The sentence should be whole, breaking it up and leaving "contacts" as a separate part of the string makes it really hard for translators to figure out the context. 

Tip: https://make.wordpress.org/core/2020/07/17/introducing-createinterpolateelement/

Reply:
but in this case I think it's ok to just add the `<strong>` tags to the translatable string

Reply:
Thanks for the suggestion—I think you're right, since we're rendering HTML via `dangerouslySetInnerHTML` anyway. I'm always a little unsure of best practices for interpolating HTML with translatable strings in React...

Reply:
@leogermani 0efabc8 updates the handling of contact counts and the translatable strings in the summaries.